### PR TITLE
improve Debian installation instructions

### PR DESCRIPTION
**Pull request by Finkes**
Signed-off-by: Dominik Finkbeiner finkes93@gmail.com

### CODE REVIEW

File: docs/sources/installation/debian.md [hunk 1]
```diff
@@ -58,8 +58,10 @@ which is officially supported by Docker.
  
         $ sudo apt-get update
         $ sudo apt-get install -t wheezy-backports linux-image-amd64
+        
+2. Restart your system so Debian will use the new kernel from wheezy-backports
```
Comment (moxiegirl) [inline docs/sources/installation/debian.md]: Missing a period on this.  While you are in there, the pattern is generally action followed by result or system response like this: 

2. Restart your system.
    This is necessary for Debian to use your new kernel.

**Comment by SvenDowideit** [comment]
LGTM - @docker/docs-owners

**Comment by moxiegirl** [comment]
@Finkes The build is failing because you did not sign your last commit.  Can  you ammend that?

https://docs.docker.com/project/set-up-prereqs/#set-your-signature-and-an-upstream-remote

**Comment by Finkes** [comment]
Can I somehow remove old commits from this pull request? the build is still failing because there are older commits with an invalid signature. Or is it good enough for the last commit to have a correct signature?

**Comment by Finkes** [comment]
Nervermind. Got it. Shouldn't crash this time.

**Comment by moxiegirl** [comment]
Super. Thanks for the signature and the fix. LGTM

**Pull request merged**